compiler/src/parsing/parsetree_iter.re

```diff
@@ -87,6 +83,7 @@ let iter_attributes = (hooks, attrs) => {
 
 let rec iter_parsed_program = (hooks, {statements} as program) => {
   hooks.enter_parsed_program(program);
+  iter_attributes(hooks, program.attributes);
```

Comment:
What's going on here?

Reply:
What do you mean? It's iterating over the attributes in the same way done with expressions and toplevels